jobs/hardware_report.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+if [[ -z "$bucket" || -z "$date" ]]; then
+    echo "Missing arguments!" 1>&2
+    exit 1
+fi
+
+# Create the package for distribution across the cluster
+git clone https://github.com/mozilla/python_mozetl.git
+cd python_mozetl
+python setup.py bdist_egg
+
+# Generate the driver script
+echo 'from mozetl.hardware_report import hardware_dashboard; hardware_dashboard.main()' > run.py
```

Comment:
This appears to be using a new(ish) idiom for python ETL jobs that is also used by churn, engagement ratio, search, and the topline reports.  That said, we're indeed missing an `env` section in the operator definition to propagate the environment variables as arguments.